config/locales/de.yml

```diff
@@ -1,4 +1,6 @@
 de:
+  date:
+    order: [ 'day', 'month', 'year' ]
```

Comment:
Why is this in English? 🇩🇪

Reply:
this is the thing I was telling you about when you said its just the "translation is missing".

I wasn't missing a normal translation, for some reason rails needed these keys for the simple_form so they have to be set

Reply:
simple_form needs the translation for attributes and the error messages (I learned this yesterday during the challenge lol)

Reply:
Ahh cool! but I think this is something else,
 because it wouldnt except anything but 'day', 'month', 'year'

Reply:
The date order `day`, `month`, `year` in the German locale configuration reflects the standard date formatting used in Germany (and many other countries outside the US). This formatting is culturally and regionally appropriate, adhering to the local customs and practices for displaying dates. In contrast to the US format, which typically follows month, day, year, many European countries, including Germany, prioritize the day before the month and the year, which is why you see it structured this way in the German translation. 🤓